apps/judicial-system/backend/migrations/20240508164452-update-case.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
+const replaceEnum = require('sequelize-replace-enum-postgres').default
+
+module.exports = {
+  async up(queryInterface) {
+    // replaceEnum does not support transactions
+    return replaceEnum({
+      queryInterface,
+      tableName: 'case',
+      columnName: 'state',
+      defaultValue: 'NEW',
+      newValues: [
```

Comment:
Did you consider changing the `state` column type to `STRING`, rather than adding a new value to an enum? I feel like we're moving away from using enums in our database tables, does that not apply to this or are we going to do it later?

Reply:
Yes I did but decided not to do it in this PR as it is somewhat risky. I have it on my list of future improvements.